kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConstants.scala

```diff
@@ -30,13 +30,14 @@ object MetricsConstants {
   final val CONN_FAIL: String = CONN + "failed"
   final val CONN_TOTAL: String = CONN + "total"
 
-  final private val STATEMENT = KYUUBI + "statement."
```

Comment:
is it safe to remove existing Metrics items directly？ cc @pan3793

Reply:
Original metric item `kyuubi.statement.total`, `kyuubi.statement.opened`, `kyuubi.statement.failed.${errorType}`, will be gone. It will be replaced with `kyuubi.operation.total.ExecuteStatement`, `kyuubi.operation.opened.ExecuteStatement`, `kyuubi.operation.failed.ExecuteStatement.${errorType}`.